sdk/search/azure-search/src/main/java/com/azure/search/SearchIndexAsyncClient.java

```diff
@@ -136,7 +137,11 @@ HttpPipeline getHttpPipeline() {
 
     /**
      * Uploads a collection of documents to the target index
-     * https://docs.microsoft.com/rest/api/searchservice/addupdate-or-delete-documents
+     * See https://docs.microsoft.com/rest/api/searchservice/addupdate-or-delete-documents
+     *
+     * Exception IndexBatchException thrown when some of the indexing actions failed, but other
+     * actions succeeded and modified the state of the index. This can happen when the Search Service is under
+     * heavy indexing load. It is important to explicitly catch this exception and check its
```

Comment:
@navalev This text is missing part of the paragraph throughout this file and `SearchIndexClient.java`. The full text (already adjusted for Java) should be:

> Exception IndexBatchException is thrown when some of the indexing actions failed, but other actions succeeded and modified the state of the index. This can happen when the Search Service is under heavy indexing load. It is important to explicitly catch this exception and check the return value of its getIndexingResult method. The indexing result reports the status of each indexing action in the batch, making it possible to determine the state of the index after a partial failure.